### ISSUE TITLE
Cannot override offline page title

### ISSUE THREAD

**Issue by Tugzrida** [gantry/gantry5#2548]
In my Grav install of Helium, I can't seem to override the offline page's title from the default "We are Down for Maintenance".

It's set in the [`user/plugins/gantry5/pages/offline.md`](https://github.com/gantry/gantry5/blob/develop/platforms/grav/gantry5/pages/offline.md) file and I can edit that file directly, but of course this change gets overridden on the next update.

I've tried overriding the file at `user/data/gantry5/themes/g5_helium/pages/offline.md` and even `user/pages/offline.md` but neither seems to work.

Taking a peek in [gantry5.php](https://github.com/gantry/gantry5/blob/develop/platforms/grav/gantry5/gantry5.php), offline.md seems to be referenced statically by its default location.

Is there some other way to override the title text or is this something that would need to be added?

**Comment by thexmanxyz** [comment]
@Tugzrida It looks like you did everything right so I'm not sure what the actual issue here is or if this is a bug. However I will [reference the override documentation](http://docs.gantry.org/gantry5/advanced/file-overrides) as well. Please check that your caches server-side and client-side (use private browser window) are cleaned. 

However I will now provide you an alternative solution that does work under Joomla:

Override the file located under:
`user/plugins/gantry5/engines/nucleus/twig.offline.html.twig`

within the folder
`user/data/gantry5/themes/THEME_DIR/engine/twig/offline.html.twig`

This might give you the desired behaviour in the meantime.

**Comment by Tugzrida** [comment]
@thexmanxyz thanks for getting back to me. The file you're talking about is at [`user/plugins/gantry5/engines/nucleus/templates/offline.html.twig`](https://github.com/gantry/gantry5/blob/develop/engines/grav/nucleus/templates/offline.html.twig) for a Grav install, however it doesn't contain the page's title.

As I mentioned, I am able to change the title by changing the [`user/plugins/gantry5/pages/offline.md`](https://github.com/gantry/gantry5/blob/develop/platforms/grav/gantry5/pages/offline.md) file (where the original title is defined) in-place, however this is not a permanent change. I haven't found a way to override this markdown file.

If the way to do this is instead by overriding the [aforementioned twig file](https://github.com/gantry/gantry5/blob/develop/engines/grav/nucleus/templates/offline.html.twig), which I am able to do, what twig do I need to add to override the HTML `<title>` tag as it isn't defined at all in this file.

**Comment by mahagr** [comment]
I think you can copy the file to `custom/[KEEP/FULL/PATH]` to make it as an override.

**Comment by Tugzrida** [comment]
@mahagr I'm aware of how to override the twig file, it's [the markdown file](https://github.com/gantry/gantry5/blob/develop/platforms/grav/gantry5/pages/offline.md) that I'm looking to override.

If that's not possible, is there some twig I could add to [`offline.html.twig`](https://github.com/gantry/gantry5/blob/develop/engines/grav/nucleus/templates/offline.html.twig) which would change the page's title?

**Comment by mahagr** [comment]
Yup, you can do the same for almost all files, not just Twig. If it does not work, let me know.

**Comment by Tugzrida** [comment]
@mahagr After reading over [this part of the docs](http://docs.gantry.org/gantry5/advanced/file-overrides#gantry-directory-structure), it would seem to me that the override path for

`GRAV_ROOT/user/plugins/gantry5/pages/offline.md`

would be

`GRAV_ROOT/user/data/gantry5/themes/g5_helium/pages/offline.md`

As I mentioned originally, I've tried this path before and I've just tried it again, clearing every single cache(and changing other details to confirm cache clearing) and the offline page title did not change according to what I put in the file at the second path above.

The docs for file overrides are somewhat confusing, but as far as I can tell, the above override path is correct according to the docs. Are you aware of a different path? As far as I can tell, a `custom` directory is only necessary for Joomla and WP.
